source/features/index.tsx

```diff
@@ -138,10 +139,10 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 });
 
 const setupPageLoad = async (id: FeatureID, config: InternalRunConfig): Promise<void> => {
-	const {include, exclude, init, deinit, additionalListeners, onlyAdditionalListeners} = config;
+	const {asLongAs, include, exclude, init, deinit, additionalListeners, onlyAdditionalListeners} = config;
 
-	// If every `include` is false and no `exclude` is true, don’t run the feature
-	if (include.every(c => !c()) || exclude.some(c => c())) {
+	// If any asLongAs is false, every `include` is false or no `exclude` is true, don’t run the feature
```

Comment:
```suggestion
	// Features are enabled if every `asLongAs` matches, at least one `include` matches, none of `exclude` matches
```

We just need to make sure that works as expected, we should probably extract this condition into a `shouldFeatureRun({include, exclude, asLongAs})` (in helpers.ts) and add tests for it.